src/configuration.ts

```diff
@@ -39,6 +39,9 @@ interface FileConfig {
 	snatchTimeout?: string;
 	searchTimeout?: string;
 	searchLimit?: number;
+	prowlarrUrl?: string;
+	prowlarrApiKey?: number;
```

Comment:
This should also be a string as well?

Reply:
Oh, right as it's a looooong number.. :D

Reply:
I think it can be all number but it can also have `[a-z]` in it as well.